recipes-nodes/rbuilder/init

```diff
@@ -52,6 +53,14 @@ start() {
     chmod 660 $RETH_DIR/db/mdbx.dat
     chmod 660 /tmp/reth.ipc
 
+    mkdir -p $RBUILDER_PERSISTENT_DIR
+    if [ ! -f $RBUILDER_PERSISTENT_DIR/rbuilder.blocklist.json ]; then
+        echo "[]" > $RBUILDER_PERSISTENT_DIR/rbuilder.blocklist.json
+        chmod 640 $RBUILDER_PERSISTENT_DIR/rbuilder.blocklist.json
+    fi
+    chown -R $RBUILDER_USER:$RBUILDER_USER /persistent/rbuilder /etc/rbuilder.config
+    chmod 640 /etc/rbuilder.config
```

Comment:
Since the rbuilder init is running almost as last service, shouldn't it be copying the fetched rbuilder.blocklist.json from somewhere else where it got rendered?
Or where are we fetching and putting the rbuilder.blocklist.json in that persistent path?

Reply:
it can be uploaded through system-api. it should not be copied around, this path seems good to point both rbuilder and system-api to.